WORKSPACE

```
@@ -300,3 +300,14 @@ http_archive(
 load("@rules_tree_sitter//tree_sitter:tree_sitter.bzl", "tree_sitter_register_toolchains")
 
 tree_sitter_register_toolchains()
+
+###############################################################################
+# bazel-diff
+###############################################################################
+
+http_jar(
+    name = "bazel_diff",
+    urls = [
+        "https://github.com/Tinder/bazel-diff/releases/download/4.3.0/bazel-diff_deploy.jar",
```

```suggestion
        "https://github.com/Tinder/bazel-diff/releases/download/4.8.0/bazel-diff_deploy.jar",
```

I assume you were copying their instructions, but 4.8 is the latest version. https://github.com/Tinder/bazel-diff/releases